x-pack/plugins/encrypted_saved_objects/server/saved_objects/index.test.ts

```diff
@@ -154,6 +156,7 @@ describe('#setupSavedObjects', () => {
         type: 'known-type',
         attributes: { attrOne: 'one', attrSecret: '*secret*' },
         references: [],
+        namespaces: ['some-ns', 'some-ns2'],
```

Comment:
Question: Isn't this test for single namespace objects?

Reply:
you are right. i have updated it.